index.bs

```diff
@@ -1508,7 +1516,38 @@ used to help facilitate isolating problems with a specific version of a device.
 If the attestation root certificate is not dedicated to a single WebAuthn Authenticator device line (i.e., AAGUID), the AAGUID
 should be specified in the attestation certificate itself, so that it can be verified against the <a>authenticatorData</a>.
 
+## Algorithm to Relax the Same-Origin Restriction ## {#algo-relax-same-origin-restriction}
+
+The same-origin policy prevents web applications from interacting, unless they both have the same origin. HTML5, via
+the `document.domain` attribute, permits "relaxing" this same-origin restriction under certain circumstances. This
+algorithm implements a similar procedure that of setting the `document.domain` attribute from [[!HTML51]], but without
+modifying the <a>Document</a> object:
+
```

Comment:
suggested build on the above 4 lines 1521-1524:

###
In general, the same-origin policy prevents web applications from interacting within user agents, unless they have the same origin. However, there are many specific facets and exceptions to this overall policy. 

For example, cookies [[RFC6265]] have same-origin restrictions similar to those of the HTML `document.domain` attribute. Both permit relaxing the same-origin restriction in a particular fashion, under certain circumstances. 

The below algorithm implements a same-origin relaxation in the same vein as cookies and `document.domain`, but without modifying the <a>Document</a> object.
###

[ for reference, see: http://identitymeme.org/http-cookie-processing-algorithm-etlds/ ]